### PR TITLE
Added Leveling

### PR DESCRIPTION
**Pull request by TheEmperor342**

### CODE REVIEW

**Comment by itsmebirdie** [comment]
Acknowledged.

File: Cogs/Miscellaneous.py [hunk 1]
```diff
@@ -91,8 +90,51 @@ async def help(self, ctx: commands.Context,*,thing=None) -> None:
             embed.add_field(name=":grin: >help Fun",value=f"```\nTo Get Help with Fun Commands\n```\n{'■'*15}\n", inline=False)
             embed.add_field(name=":o2: >help Miscellaneous", value=f"```\nTo Get Help with Other Commands\n```\n{'■'*15}\n", inline=False)
             embed.add_field(name=":notepad_spiral: >help Embeds", value=f"```\nTo Get Help with Embed Commands\n```\n{'■'*15}", inline=False)
+            embed.add_field(name=":musical_note: >help Music", value=f"```\nTo Get Help with Music Commands\n```\n{'■'*15}", inline=False)
+            embed.add_field(name=f"""{choice([":laughing:", ":grinning:", ":smiley:", ":smile:", ":grin:", ":laughing:"])} >help LON""", value=f"```\nTo Get Help with LON Commands\n```""", inline=False)
```
Comment (itsmebirdie) [inline Cogs/Miscellaneous.py]: I guess help command of music is already made as per #59 [Changes in Music.py and Miscelaneous.py commands](https://github.com/hackarmour/discord-assistant/pull/59)

File: README.md [hunk 2]
```diff
@@ -13,12 +13,14 @@ ___
...
 > [TheEmperor342](https://github.com/TheEmperor342)\
 > [PhantomKnight287](https://github.com/PhantomKnight287)
+> [Hrithik326](https://github.com/Hrithik326)
 ___
 
 ## Requirements
 - discord.py == 1.7.3
 - dpymenus == 2.1.5
 - requests == 2.25.1
+- youtube_dl == 2021.6.6
```
Comment (itsmebirdie) [inline README.md:23]: Adding `ffmpeg` as an external package requirement is recommended.

Comment (itsmebirdie) [reply README.md:23]: Resolved in commit [#af1e6bf0f89e0ff66e8451cf58707497823e3d87](https://github.com/hackarmour/discord-assistant/pull/61/commits/af1e6bf0f89e0ff66e8451cf58707497823e3d87)

File: main.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 TOKEN = input("Enter The Token of your bot: ")
 
 # Initialize the bot instance
-bot = commands.Bot(command_prefix=">", intents=discord.Intents.all(),case_insensitive=True)
+bot = commands.Bot(command_prefix="!", intents=discord.Intents.all(),case_insensitive=True)
```
Comment (itsmebirdie) [inline main.py:10]: wrong prefix

Comment (itsmebirdie) [reply main.py]: [resolved](https://github.com/hackarmour/discord-assistant/pull/61/commits/40746aff112d7647606daeb80d92e7874dd0ad62)

File: Cogs/Miscellaneous.py (hunk 1 above)
Comment (TheEmperor342) [reply Cogs/Miscellaneous.py]: do i fix that?

File: Configuration/Leveling.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        "Enabled": true,
+        "Levels": {
+            "754894159403286531": [
+                91,
+                2,
+                291
+            ]
+        }
+    }
+}
```
Comment (itsmebirdie) [inline Configuration/Leveling.json:17]: you accidentally committed the testing bot's actual json...

**Comment by TheEmperor342** [comment]
I have to update the help as well so please wait

**Review by itsmebirdie — COMMENTED** [review]
Code review done. Awaiting testing

File: Configuration/Leveling.json (hunk 4 above)
Comment (TheEmperor342) [reply Configuration/Leveling.json:17]: i will fix this in the next commit

File: Cogs/Miscellaneous.py (hunk 1 above)
Comment (itsmebirdie) [reply Cogs/Miscellaneous.py]: > do i fix that?

yes, please do

**Review by itsmebirdie — APPROVED** [review]
Reviewed and tested.
Approved for merge 😃

**Pull request merged**